deploy/clowdapp.yaml

```diff
@@ -826,3 +828,6 @@ parameters:
   - name: PULP_STATIC_URL
     description: The path to the static assets needed by some Django apps
     value: "/api/pulp/assets/"
+  - name: PULP_MIDDLEWARE
+    description: List of Middleware
+    value: "@merge pulp_service.app.middleware.ProfilerMiddleware"
```

Comment:
Are we sure this will be earliest in the middleware?

Reply:
It will be the last middleware in the list.

Reply:
That's better than not in the list! Is there a way to make it first? I'm interested in investigating the basic auth middleware overhead for example. If not I'm ok with trying this as-is also.